tgui/packages/tgui/components/RoundGauge.js

```diff
@@ -23,6 +23,7 @@ export const RoundGauge = props => {
     maxValue = 1,
     ranges,
     alertAfter,
+    alertBefore,
```

Comment:
Is this included in the upstream tgui component? If not, possibly PR in there (and link here for a use-case)? Prefer not deviating from upstream tgui for core components if possible.

Reply:
You know I didn't even think about the upstream. I'll PR it.

Reply:
So fun fact, I went to go do this and it's already done in the upstream in almost exactly the way I did it.